### DISCUSSION TITLE
OpenRadioss On windows - Issues

### DISCUSSION THREAD

**Discussion by IssamIzzo** [OpenRadioss/OpenRadioss#1534] [Users]
Hello everyone ! 
I hope that you're doing great ! 

I'm a student, and recently, I tried to write as a user in invite commandes (OpenRadioss on Windows), but I couldn't, It stops at *Run engine*, & then it writes 'The specified path was not found'. Should I really us the invite commandes as an administrator ?

Does anyone know how can I solve this problem ?

Thank you in advance !

**Comment by MarianBulla** [comment]
Hello, 
and welcome to the OpenRadioss community!

No you don't need to be an administrator to be able to start OpenRadioss models on windows.
You just need to set all the necessary environment variables, as described on the "How to run OpenRadioss" page https://github.com/OpenRadioss/OpenRadioss/blob/main/INSTALL.md

What are the steps, you're doing to start OpenRadioss ?
(need more details and/or a screenshot to be able to provide you a more detailled help)

Thanks and best regards,
Marian

**Reply by IssamIzzo** [reply]
Hello Marian,
First thank you for your return,

Thats some good News! Please find attached a screenshot of the full process that I used to run my exemple,

Thank you for your help,

Best regards,

![image](https://github.com/OpenRadioss/OpenRadioss/assets/135626307/b6ff9651-2533-430b-8e26-645e883b7482)

**Comment by MarianBulla** [comment]
Hello,
here is what I do, if I want to use the pre-compiled executables, of OpenRadioss:
1. Checking if everything is unzipped in a folder (path without any special characters, like balncs/spaces etc.): 
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/b1c8dc28-31b2-447e-9551-6085a8ab1dec)

2. Setting all nexeccary environment variables (this step can be automated, using a batch file or the graphical user interface...you can find in this discussion board, provided by Paul):
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/257b299c-46ff-4ccc-ac33-d2195551c4c1)

3. Executing the starter with the model file:

![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/3b1def1d-d1f5-4126-a85f-5f05c7188487)

4. Executing the engine (=RADIOSS solver), using 8 cpus:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/f9a976c0-47f2-4d71-bfbb-75dbb72fac94)


Hope it helps you ?

Thanks and best regards,
Marian

**Reply by IssamIzzo** [reply]
Hello,

I've tried you're methode with adding all the missing  environment variables settings, but I have the following problem ? 
![image](https://github.com/OpenRadioss/OpenRadioss/assets/135626307/2cda6808-c886-4b43-af16-cb4b9eee71c4)


The .dll files can't be read, but I still cant understand why it's not working even if all the files are unzipped, do you have any ideas ? 

Thank you so much for you're valuable help !

**Comment by MarianBulla** [comment]
Hello,

The OpenaRadioss starter is not able to find the necessary libraries.

Can you add the "set ...." envornment variables, please:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/3042ad7d-76fa-4125-84b8-df65da10cb2f)

And try again.

Thanks and best regards
Marian

**Reply by IssamIzzo** [reply]
Hello Mr. Marian,

After trying your methodology & procedure, OpenRadioss is succesfully working now, At the end it's the environnement settings who made all the issues that I had.

Thank you so much for you help & for your support !

Have a great day !

**Comment by MarianBulla** [comment]
Hello Issamizzo,

Great to hear that you are able to run OpenRadioss, now.
Also glad beeing able to help you.

Enjoy "playing" with OpenRadioss :-)

Thanks and best regards,
Marian
